tests/commands/test_commands.py

```diff
@@ -714,9 +716,7 @@ def test_get_ui_download_url_direct(mocker):
             },
         ]
     )
-    get_mock = mocker.patch(
-        "freqtrade.commands.deploy_commands.requests.get", return_value=response
-    )
+    get_mock = mocker.patch("requests.get", return_value=response)
```

Comment:
is requests really this heavy a dependency? 

as mentioned elsewhere - patching "Into" 3rd party modules is something we should avoid - and as far as i'm aware - we're currently (develop branch) not doing anywhere.

Reply:
```
import time
a = time.time()
import requests
b = time.time()
import questionary
c = time.time()
print(b - a, c - b)
# 0.11817073822021484 0.07719016075134277
```
No, it's hard to resist the temptation of 100% increase, even if it's not much beneficial. Haha

Reply:
It shows `requests` accounting for more time than `questionary`, so we have no reason to let it go after getting rid of `questionary`. What's your suggestion about the test? It looks strange to me if we indirectly import `requests`  as it's so commonly used.

Reply:
well we can accomplish this rather easily by moving stuff around a bit (see my commits).